scripts/mktest.py

```diff
@@ -0,0 +1,277 @@
+import requests
+import xml.etree.ElementTree as ET
+import tilequeue.tile as tile
+from ModestMaps.Core import Coordinate
+from jinja2 import Template
+import os
+from contextlib import contextmanager
+
+
+def way_full(way_id, zoom=16):
+    url = 'https://api.openstreetmap.org/api/0.6/way/%d/full' % (way_id,)
```

Comment:
nit: just `% way_id` would work here

Reply:
If `way_id` somehow becomes a tuple then we get an error saying "not all arguments converted during string formatting", rather than "a number is required, not tuple". Generally, this isn't going to happen... but I've shot myself in the foot that way a number of times! And I've found it terribly confusing. So now I try to default to using `% (thing,)` when I think there should only be one thing.